projects/simulation/SMPL+D_body_models/README.md

```diff
@@ -0,0 +1,60 @@
+# SMPL+D body models
+
+This folder contains code for downloading a large number of human models in SMPL+D body model format, as well as code for making them able to be transferred to Webots for simulations.
+
+<p float="left">
+  <img src="./examples/model_1.png" width=150 />
+  <img src="./examples/model_4.png" width=150 />
+  <img src="./examples/model_3.png" width=150 />
+  <img src="./examples/model_2.png" width=150 />
+</p>
+
+### Download the raw SMPL+D models only.
+```
+python download_data.py raw
+```
+
+### Download the SMPL+D models and convert them as FBX (Filmbox) files. <br/>In this case the pose blend shapes are not applied.
+
+- Download the data by running:

```

Comment:
Should probably mention how big this file is so users are aware.

Reply:
> Should probably mention how big this file is so users are aware.

I added their size in the description